build/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../tsconfig-base.json",
+  "compilerOptions": {
+    "outDir": "../.tmp/tsbuildinfo/build",
+  },
+  "references": [
+    {"path": "../"},
+    {"path": "../treemap/"},
+  ],
+  "include": [
+    "**/*.js",
+    "../types/global-lh.d.ts",
+  ],
+}
```

Comment:
@brendankenny any notes?

Reply:
What's it for? I'm surprised anything changed

Reply:
we import treemap ui strings and the locales module in build-treemap.js

and that requires various types to exist on the global scope otherwise tsc fails

Reply:
FWIW `{"path": "./treemap/"},` in the root `tsconfig.json` would have also fixed it, though it's not necessarily bad to have a tsconfig for this folder too